subprojects/degrees-and-programs/source/cascade/formats/Degrees-programs/PrimaryContent.vm

```diff
@@ -272,13 +269,15 @@
 
 										#foreach ( $motivation in $motivations )
 
+											#set ( $rand = $_MathTool.floor( $_MathTool.mul( $_MathTool.getRandom(), 10000 ) ) )
```

Comment:
A randomly generated number helps create a unique identifier for each of the interest's corresponding label and input field in order to avoid conflict with other interests' labels and input fields.

For example, we have two interests with a value of "Science & Technology." These values are used and mutated to help create an id for the input field: E.g. "dap-discover-interest-${rand}-science-and-technology". However, without a randomly generated number in conjunction, ids will be duplicated when values are similar and create a conflict with Accessibility.

Browsers are forgiving when rendering HTML, but duplicate ids can create potential problems with behavior as well. So, this addition will help prevent any issues with JavaScript in the future.